src/config/dynamic-theme-fixes.config

```diff
@@ -25441,8 +25469,6 @@ wikiless.org
 *.wikipedia.org
 
 INVERT
-.mwe-math-fallback-image-inline
-.mwe-math-fallback-image-display
```

Comment:
Correct selector for img element, see https://github.com/darkreader/darkreader/issues/12001#issuecomment-1879746162